<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductForm</class>
 <widget class="QDialog" name="ProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../app/icons.qrc">
    <normaloff>:/new</normaloff>:/new</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;
font: 8pt &quot;Verdana&quot;;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>141</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>New Product</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>163</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>741</width>
     <height>16</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableView" name="product_view">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>318</y>
     <width>721</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>721</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>721</width>
     <height>241</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>274</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Products On File</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>296</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>741</width>
     <height>16</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="delete_tool_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>19</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/delete</normaloff>:/delete</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="cancel_tool_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>580</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/cancel</normaloff>:/cancel</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>188</y>
     <width>658</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>658</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>658</width>
     <height>48</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Manufacturer:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="manufacturer_line_edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Category:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="category_line_edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Model:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="model_line_edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Capacity:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="capacity_line_edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="color_line_edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="add_product_tool_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>107</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>107</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>107</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/new</normaloff>:/new</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>781</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:'#FF7F7F'</string>
   </property>
   <property name="title">
    <string>RULES:</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Manufacturer, category, model and color must not contain spaces. Use '-' or '_'. These can be removed later when  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity must be a natural number. Ex.: 12, 16, 4453, 12341234, 256. Not decimal number.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>creating invoices. Ex.: Space-gray, midnight-green, light_gold</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>manufacturer_line_edit</tabstop>
  <tabstop>category_line_edit</tabstop>
  <tabstop>model_line_edit</tabstop>
  <tabstop>capacity_line_edit</tabstop>
  <tabstop>color_line_edit</tabstop>
  <tabstop>add_product_tool_button</tabstop>
  <tabstop>product_view</tabstop>
  <tabstop>delete_tool_button</tabstop>
  <tabstop>cancel_tool_button</tabstop>
 </tabstops>
 <resources>
  <include location="../app/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_tool_button</sender>
   <signal>clicked()</signal>
   <receiver>ProductForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
